### ISSUE TITLE
Can the rangeslider be positioned both above or below the graph?

### ISSUE THREAD

**Issue by terrylucas** [plotly/plotly.js#3543]
The rangeslider is great.  Is it possible to place to the rangeslider above the chart instead of below (using CSS or some other attribute mechanism)?  I haven't found anything in the docs that mentions this type of functionality feature.  I can change the transform to translate the y position but that's not quite right (seems to be recomputed).

Thanks.

**Comment by etpinard** [comment]
Duplicate of https://github.com/plotly/plotly.js/issues/2453

**Comment by terrylucas** [comment]
I noticed this issue was marked as a duplicate of an issue asking for a y-axis rangeslider, I'm not asking for that - but the positioning of an x-axis rangeslider above the chart instead of below the chart.

I did attempt to change the transform attribute of the `rangeslider-contain` however, plotly.js must  compute / change the transform with each extendTrace.  I was able to change the transform attribute from:

`transform="translate(80,129)"
`

to (80, 0) which will place the slider at the top of the chart.  But each extendTrace the transform again becomes:

`transform="translate(80,129)"
`
Is this a request that that's a won't fix?
